### PR TITLE
TEST: Increase coverage on game_theory

### PR DESCRIPTION
**Pull request by QBatista**
Adds tests to increase the coverage on game_theory

### CODE REVIEW

File: quantecon/game_theory/tests/test_mclennan_tourky.py [hunk 1]
```diff
@@ -54,13 +58,102 @@ def test_pure_nash(self):
...
+
+    def test_is_epsilon_nash_no_indptr(self):
+        ok_(_is_epsilon_nash([1., 0., 0., 1., 0.], self.g, 1e-5))
+
+
+def test_flatten_action_profile():
+    unflattened_actions = [[1/3, 1/3, 1/3], [1/2, 1/2]]
+    flattened_actions = [1/3, 1/3, 1/3, 1/2, 1/2]
+    test_obj = _flatten_action_profile(unflattened_actions, [0, 3, 5])
+    assert_allclose(test_obj, flattened_actions)
```
Comment (oyamad) [inline quantecon/game_theory/tests/test_mclennan_tourky.py]: Shouldn't this be `assert_array_equal`?

File: quantecon/game_theory/tests/test_mclennan_tourky.py [hunk 2]
```diff
@@ -5,8 +5,12 @@
 
 """
 import numpy as np
-from nose.tools import ok_
+from numpy.testing import assert_allclose
+from nose.tools import ok_, raises
 from quantecon.game_theory import Player, NormalFormGame, mclennan_tourky
+from quantecon.game_theory.mclennan_tourky import _best_response_selection, \
+                                                  _flatten_action_profile, \
+                                                  _is_epsilon_nash
```
Comment (oyamad) [inline quantecon/game_theory/tests/test_mclennan_tourky.py]: Maybe this should be something like

```py
from quantecon.game_theory.mclennan_tourky import (
    _best_response_selection, _flatten_action_profile, _is_epsilon_nash
)
```

File: quantecon/game_theory/tests/test_mclennan_tourky.py [hunk 3]
```diff
@@ -54,13 +58,102 @@ def test_pure_nash(self):
...
+def test_best_response_selection_no_indptr():
+    bimatrix = [[(3, 3), (3, 2)],
+                [(2, 2), (5, 6)],
+                [(0, 3), (6, 1)]]
+    g = NormalFormGame(bimatrix)
+
+    test_obj = _best_response_selection([1/3, 1/3, 1/3, 1/2,1/2], g)
+    expected_output = np.array([0., 1., 0., 0., 1.])
+
+    assert_allclose(test_obj, expected_output)
```
Comment (oyamad) [inline quantecon/game_theory/tests/test_mclennan_tourky.py]: Shouldn't this be `assert_array_equal`?

File: quantecon/game_theory/tests/test_mclennan_tourky.py (hunk 2 above)
Comment (mmcky) [reply quantecon/game_theory/tests/test_mclennan_tourky.py]: thanks @oyamad I didn't know you could use ``(`` like this.

**Comment by QBatista** [comment]
@oyamad This is ready.

**Comment by mmcky** [comment]
Thanks @QBatista and @oyamad

**Pull request merged**